jwst/resample/tests/test_utils.py

```diff
@@ -118,7 +118,8 @@ def test_build_driz_weight(weight_type):
     """Check that correct weight map is returned of different weight types"""
     model = ImageModel((10, 10))
     model.dq[0] = DO_NOT_USE
-    model.meta.exposure.exposure_time = 10.0
+    # model.meta.exposure.exposure_time = 10.0
```

Comment:
Does this really need to be commented out? It may no longer be used, but it doesn't hurt to set it anyway in the model that's being created. If you really don't want it in the model anymore, then it's probably better to just delete the line, rather than commenting.